datafusion/expr/src/expr_fn.rs

```diff
@@ -215,6 +215,18 @@ pub fn count(expr: Expr) -> Expr {
     ))
 }
 
+/// Create an expression to represent the COUNT(IS NULL x) aggregate function
```

Comment:
I found this confusing as first because it isn't `COUNT(x IS NULL)` -- I think a more accurate description is `COUNT(x) FILTER (WHERE x IS NULL)` or something like that 

```suggestion
/// Create an aggregate that returns the number `NULL` values in a column
```